leaf-ui/css/style.css

```diff
@@ -45,6 +45,36 @@ body.waiting *
   font-style: italic;
 }
 
+/** experimental dim screen */ 
+.dimScreen
+{
+    position:fixed;
+    padding:0;
+    margin:0;
+
+    top:0;
+    left:0;
+
+    width: 100%;
+    height: 100%;
+    background:rgba(255,255,255,0.5);
+}
+
+/** Dims the entire page **/ 
```

Comment:
Might want to add some comments like the previous ones

Reply:
Done